openedx/features/smart_referral/tests/test_helpers.py

```diff
@@ -0,0 +1,122 @@
+# coding=UTF-8
+"""Tests for helpers methods of smart referral."""
+import mock
+
+from ddt import ddt, file_data
+from django.test import TestCase
+from lms.djangoapps.onboarding.tests.factories import OrganizationFactory, UserFactory
+
+from .constants import (
+    SORTED_PLATFORM_CONTACT_LIST_KEY,
+    SORTED_PLATFORM_CONTACT_LIST_INDEX,
+    SORTED_NON_PLATFORM_CONTACT_LIST_KEY,
+    SORTED_NON_PLATFORM_CONTACT_LIST_INDEX,
+    SORTED_CONTACT_LIST_KEY,
+    SORTED_CONTACT_LIST_INDEX,
+    UNSORTED_CONTACT_LIST_KEY,
+    UNSORTED_CONTACT_LIST_INDEX,
+)
+from .. import helpers as filter_contacts_helpers
+
+
+@ddt
+class FilterContactsAPIViewTestHelpers(TestCase):
+
+    def setUp(self):
```

Comment:
do we really need to override this method if we are just calling the parent method in it?